CONTRIBUTING.md

```diff
@@ -117,17 +117,23 @@ To watch the logs of this IntelliJ instance, run `tail -f build/idea-sandbox/sys
 
 ## Run UI tests
 
-```
+```shell
 ./gradlew [-Pagainst=<e.g. 2021.2>] [-Pheadless] [-Ptests=<e.g. toggle>] uiTest
 ```
 
 See [Robot plugin](https://github.com/JetBrains/intellij-ui-test-robot)
 and [a preso on testing UI of IntelliJ Plugins](https://slides.com/plipski/intellij-plugin-ui-testing) for more details.
 
 
-## Check dependency updates
+## Update version catalog
 
-`./gradlew dependencyUpdates`
+```shell
+./gradlew versionCatalogUpdate
```

Comment:
This updates `gradle/libs.versions.toml`.

`dependencyUpdates` only printed out the new versions, to be manually updated ☹️